.travis.yml

```diff
@@ -60,6 +60,12 @@ install:
       pip install -U wheel
     fi
 
+    if [ "$ENV" = "testvim" ]; then
+      set -x
+      python -V
+      python2 -V
+    fi
+
```

Comment:
Debugging I guess?  Should be removed.
But might be useful to have/keep in general?!  What do you think?
(I've added it in ac84dc8ac)